core/storage/feedback/gae_models.py

```diff
@@ -504,8 +504,8 @@ class GeneralFeedbackThreadUserModel(base_models.BaseModel):
     Instances of this class have keys of the form [user_id].[thread_id]
     """
 
-    user_id = ndb.StringProperty(required=False, indexed=True)
-    thread_id = ndb.StringProperty(required=False, indexed=True)
+    user_id = ndb.StringProperty(required=True, indexed=True)
```

Comment:
This could break existing models, right? Feedback threads can be started anonymously.

Reply:
No, this was fixed in #8481. Anonymous models don't have `GeneralFeedbackThreadUserModel`.